package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "npm run clean && babel src -d lib",
     "clean": "rimraf lib",
-    "prepublish": "npm run build",
+    "prepublish": "in-publish && safe-publish-latest && npm run build || not-in-publish",
     "lint": "eslint --ext .js,.jsx src test stories",
```

Comment:
we're going to need
```
"prelint": "npm run build",
```
I *think*

Reply:
Actually let's just fix the import in `geometric-test.js` to use the src file not the built file